docs/pages/toolpad/index.js

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
+import { demos, docs, demoComponents } from '../../data/toolpad/index.md?@mui/markdown';
```

Comment:
Would this work? Could be simpler than having to think about relative paths.

```suggestion
import { demos, docs, demoComponents } from 'docs/data/toolpad/index.md?@mui/markdown';
```

Reply:
This would conflict with the `docs` alias that we use to fetch files from the monorepo dependency. We could potentially create another alias called `docsToolpad`, but that would conflict with the pattern of only using relative paths in the rest of the project

Reply:
Ah right, I forgot about this. on MUI X, they have a `docsx` alias.